File: 프로그래머스/unrated/181934. 조건 문자열/조건 문자열.java
class Solution {
   public int solution(String ineq, String eq, int n, int m) {
      int ans = -1;

      switch (ineq + eq) {
         case ">=" -> ans = n >= m ? 1 : 0;
         case ">!" -> ans = n > m ? 1 : 0;
         case "<=" -> ans = n <= m ? 1 : 0;
         case "<!" -> ans = n < m ? 1 : 0;
      }

      return ans;
   }
}

File: 프로그래머스/unrated/181934. 조건 문자열/조건 문자열.kt
class Solution {
    fun solution(ineq: String, eq: String, n: Int, m: Int): Int {
        var ans = 0
        when(ineq + eq) {
            ">=" -> ans = if (n >= m) 1 else 0
            ">!" -> ans = if (n > m) 1 else 0
            "<!" -> ans = if (n < m) 1 else 0
            "<=" -> ans = if (n <= m) 1 else 0
        }

        return ans
    }
}